### ISSUE TITLE
Getting a weird crash in 0.15

### ISSUE THREAD

**Issue by ethereumdegen** [komadori/bevy_mod_outline#49]
```
thread '<unnamed>' panicked at /home/andy/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_mod_outline-0.9.0/src/flood/compose_output.rs:209:14:
called `Option::unwrap()` on a `None` value
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Encountered a panic in system `bevy_render::renderer::render_system`!
2024-12-16T06:10:23.169561Z  INFO broken_spirit::terrain: waiting for terrain root entity
2024-12-16T06:10:23.169999Z  WARN broken_spirit::level_instantiation::spawning::objects::unit: no terrain viewers
thread 'main' panicked at /home/andy/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-hal-23.0.1/src/vulkan/instance.rs:173:58:
Trying to destroy a SurfaceSemaphores that is still in use by a SurfaceTexture


```

**Comment by komadori** [comment]
I assume you're using `OutlineMode::FloodFlat`, since this panic is in the jump flood code.

It looks like the expected way to use `get_render_pipeline()` is to just skip the RenderNode for that frame if it fails. Fixed in 3bd8357c656f6100c3fe13a2d6bab50f6de72a6f. Does this resolve your issue?

I will make a new minor release soon.

**Comment by ethereumdegen** [comment]
yes that helped
